tests/integration/security/sds_ingress/util/util.go

```diff
@@ -88,7 +90,7 @@ var IngressCredentialServerKeyCertB = IngressCredential{
 // and creates K8s secrets for ingress gateway.
 // nolint: interfacer
 func CreateIngressKubeSecret(t test.Failer, ctx framework.TestContext, credNames []string,
-	ingressType ingress.CallType, ingressCred IngressCredential) {
+	ingressType ingress.CallType, ingressCred IngressCredential, isGenericSecret bool) {
```

Comment:
Change isGenericSecret to isCompoundGeneric, to make it clear that this flag is only used for creating compound secret.

Reply:
i think we should not set it specific for compound secret. As many other functions use this flag. How about I set a new flag called the isCompoundWithCaCert to specify for that test